modules/meeting/app/components/meeting_agenda_items/item_component/show_component.sass

```
@@ -63,3 +63,9 @@ $meeting-agenda-item--presenter-width--mobile: 150px
 
     .custom-presenter-flex
       margin-left: 0
+
+// Highlight agenda item when linked to it
+.op-meeting-agenda-item-wrapper
+  &:target
+    border: 2px solid var(--borderColor-accent-emphasis)
```

I'd prefer that to be an outline to avoid that visual jump when clicking outside of it.

```suggestion
    outline: 2px solid var(--borderColor-accent-emphasis)
```

---

I'm fine with that, but we should adapt the AC then, and ideally use the same for work packages?  /cc @judithroth